### ISSUE TITLE
Issue while dragging and scrolling any object.

### ISSUE THREAD

**Issue by nishithzaverivt** [fatlinesofcode/ngDraggable#234]
I have faced issue of different coordinates of cursor and object/image when using cloning functionality of ngDraggable in latest stable version (0.1.8).

Issue is observed when we scroll down the page and after that drag and drop image. But in case of ngDraggable from master branch this issue is not reproducible.

I have attached screen shot of ngDraggable example with latest stable release(0.1.8) and master branch. 

I want to use functionality which is working in Master branch but not working in 0.1.8. So I request to create new version which includes fix of this issue.

**0.1.8 Release Screen Shot.**

![stable version 0 1 8](https://cloud.githubusercontent.com/assets/16607267/13943282/f7e4097c-f020-11e5-8f56-46c33ec5487f.png)

**Master Branch Screen Shot.**

![master branch](https://cloud.githubusercontent.com/assets/16607267/13943281/f7dd05aa-f020-11e5-95a0-8c61316f0dbf.png)

**Comment by sondun2001** [comment]
Yeah, there is an outstanding pull request, but doesn't seem to fix it. This one is really bad...

**Comment by mhchristensen** [comment]
I also see this issues.  I believe its related to fixed positioning combined with the matrix transform
